src/packagedcode/plugin_package.py

```diff
@@ -364,7 +391,7 @@ def create_package_and_deps(codebase, package_adder=add_to_package, strip_root=F
         **kwargs
     )
 
-    codebase.attributes.packages.extend(package.to_dict() for package in packages)
+    codebase.attributes.packages.extend(package.to_dict(package_summary) for package in packages)
```

Comment:
```suggestion
    codebase.attributes.packages.extend(
       package.to_dict(package_summary)
       for package in packages
    )
```

Format these for readability.